bleach/__init__.py

```diff
@@ -43,6 +44,8 @@
 
 ALLOWED_STYLES = []
 
+ALLOWED_PROTOCOLS = copy.copy(HTMLSanitizer.acceptable_protocols)
```

Comment:
Given that we're now "owning this" and we want to document it explicitly, I think it's prudent that instead of copying it with `copy.copy`, that we make it explicitly defined here:

```
ALLOWED_PROTOCOLS = [u'ed2k', u'ftp', u'http', u'https', u'irc', u'mailto', u'news', u'gopher', u'nntp', u'telnet', u'webcal', u'xmpp', u'callto', u'feed', u'urn', u'aim', u'rsync', u'tag', u'ssh', u'sftp', u'rtsp', u'afs', u'data']
```

That way our list won't change between versions of html5lib and bleach can explicitly declare what we think is appropriate regardless of what html5lib says.

Related to this, I'm not sure I'd consider that list a list of "safe protocols". I think I'd want to trim it down to a much smaller subset. Maybe this much more conservative set:

```
ALLOWED_PROTOCOLS = [u'http', u'https', u'mailto']
```

Anyone have thoughts on that?